### DISCUSSION TITLE
Global Search

### DISCUSSION THREAD

**Discussion by dev-dhull** [filamentphp/filament#3140] [Help] [answered]
When I use the global search function when I search for any type of data. I faced this type of error.
can u said me how to resolve this error :->
Property [$searchQuery] not found on component: [filament.http.livewire.global-search]

**Comment by danharrin** [comment, marked as answer]
1) `php artisan filament:upgrade`
2) Delete `/resources/views/vendor` if it exists

**Reply by dev-dhull** [reply]
thanks
